tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -131,9 +131,11 @@ private[scalanative] object LLVM {
       s"Linking native code (${config.gc.name} gc, ${config.LTO.name} lto)"
     ) {
       config.logger.running(compile)
-      Process(compile, config.workdir.toFile) ! Logger.toProcessLogger(
-        config.logger
-      )
+      val result = Process(compile, config.workdir.toFile) ! Logger
+        .toProcessLogger(config.logger)
```

Comment:
This reads very weirdly. Can you convince scalafmt to format it as
```suggestion
      val result = Process(compile, config.workdir.toFile) !
        Logger.toProcessLogger(config.logger)
```
?

Reply:
Yes, that's passible. Fixed